app/instance-initializers/axe-component.js

```diff
@@ -8,27 +10,32 @@ let hasRan = false;
 export function initialize(application) {
   if (hasRan) { return; }
 
+  const appConfig = ENV['ember-a11y-testing'] || {};
+  const { componentOptions: { axeOptions, turnAuditOff, axeCallback } = {} } = appConfig;
+
   Ember.Component.reopen({
     /**
      * An optional callback to process the results from the a11yCheck.
      * @public
      * @type {Function}
      */
-    axeCallback: undefined,
+    axeCallback,
 
     /**
      * An optional options object to be used in a11yCheck.
      * @public
      * @type {Object}
      */
-    axeOptions: undefined,
+    axeOptions,
 
     /**
      * Turns off the accessibility audit during rendering.
+     * Defaults to false if not set in the application's configuration.
+     *
      * @public
      * @type {Boolean}
      */
-    turnAuditOff: false,
+    turnAuditOff: typeof turnAuditOff === 'boolean' ? turnAuditOff : false,
```

Comment:
The configuration of which rules you enable inside of aXe is probably the way to go for this sort of behavior instead of turning off testing for all components. Turning on any linting or automated testing approach in an existing project is hard (and I've experienced that pain) and I don't mind the "surprise, you've got a lot of work to do!" because I don't think it's particularly surprising.

I don't want to have to set up a linter on top of this which checks to make sure you're not doing the wrong thing with your project config.